src/sentry/static/sentry/app/views/settings/incidentRules/constants.tsx

```diff
@@ -1,48 +1,96 @@
+import {t} from 'app/locale';
 import {
   AlertRuleThresholdType,
   UnsavedIncidentRule,
   Trigger,
   Dataset,
 } from 'app/views/settings/incidentRules/types';
+import {Incident} from 'app/views/alerts/types';
+import {NewQuery} from 'app/types';
 
 export const DEFAULT_AGGREGATE = 'count()';
 
-export const PRESET_AGGREGATES = [
+type Preset = {
+  /**
+   * The regex used to match aggregates to this preset.
+   */
+  match: RegExp;
+  /**
+   * The name of the preset
+   */
+  name: string;
+  /**
+   * The dataset that this preset applys to.
+   */
+  validDataset: Dataset[];
+  /**
+   * The default aggregate to use when selecting this preset
+   */
+  default: string;
+  /**
+   * Used to generate additional parameters in the 'show in discover' button on
+   * the alert details page.
+   */
+  makeQueryParameters: (incident: Incident) => Partial<NewQuery>;
+};
+
+export const PRESET_AGGREGATES: Preset[] = [
   {
     match: /^count\(\)/,
-    name: 'Number of errors',
+    name: t('Number of errors'),
     validDataset: [Dataset.ERRORS],
     default: 'count()',
+    makeQueryParameters: (incident: Incident) => ({
+      fields: ['issue', incident.alertRule.aggregate],
+    }),
   },
   {
     match: /^count_unique\(tags\[sentry:user\]\)/,
-    name: 'Users affected',
+    name: t('Users affected'),
     validDataset: [Dataset.ERRORS],
     default: 'count_unique(tags[sentry:user])',
+    makeQueryParameters: (incident: Incident) => ({
+      fields: ['issue', incident.alertRule.aggregate],
+    }),
   },
   {
     match: /^(p[0-9]{2,3}|percentile\(transaction\.duration,[^)]+\))/,
-    name: 'Latency',
+    name: t('Latency'),
     validDataset: [Dataset.TRANSACTIONS],
     default: 'percentile(transaction.duration, 0.95)',
+    makeQueryParameters: (incident: Incident) => ({
+      fields: ['id', 'transaction.duration', incident.alertRule.aggregate],
```

Comment:
If you include `id` in a discover query with aggregates you don't generally get useful results as the aggregate is only applied to the single row.